sway-core/src/asm_lang/allocated_ops.rs

```diff
@@ -594,6 +594,7 @@ impl AllocatedOp {
 
             /* Memory Instructions */
             ALOC(a) => op::ALOC::new(a.to_reg_id()).into(),
+            CFEI(a) if a.value == 0 => op::NOOP::new().into(),
```

Comment:
Shouldn't the same thing be done for `CFSI` also?

Reply:
will do